resources/modules/custom/MODULESHORTHAND_media/MODULESHORTHAND_media.info

```diff
@@ -1,6 +1,6 @@
-name = Reload Media
+name = Media settings
```

Comment:
instead of just removing Reload.. Should we maybe prefix site? I.E. "Site media settings"?